macros/CSSRef.ejs

```diff
@@ -17,12 +17,12 @@ if (slug) {
     var htmlEscape = kuma.htmlEscape;
     var rtlLocales = ['ar', 'he', 'fa'];
 
-    var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/properties.json";
-    var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/at-rules.json";
-    var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/selectors.json";
-    var typesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/types.json";
-    var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/syntaxes.json";
-    var unitsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/units.json";
+    var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/properties.json";
+    var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/at-rules.json";
+    var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/selectors.json";
+    var typesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/types.json";
+    var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/syntaxes.json";
+    var unitsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/units.json";
```

Comment:
There is code like `var pageList = page.subpagesExpand('/en-US/docs/Web/CSS');`. A method that calls the `$children?expand` endpoint of Kuma. The `pageList` variable is empty for me, so something is wrong locally with this. I do have a few subpages and http://localhost:8000/en-US/docs/Web/CSS$children?expand works as expected. Maybe caching in KS, I don't know how to fix that. I might just work fine on staging.